### PR TITLE
Docs add docs for flutter auth UI

### PR DESCRIPTION
**Pull request by FatumaA**
## What kind of change does this PR introduce?

Update auth helpers doc to include the flutter auth helper.

## What is the current behavior?

The flutter auth helper was not documented.

## What is the new behavior?

It is now documented as part of the auth helpers.

## Additional context
related to this [issue](https://github.com/supabase-community/flutter-auth-ui/issues/16).

Screenshot 2023-06-06 at 16 59 15

### CODE REVIEW

**Comment by FatumaA** [comment]
@dshukertjr you can take a look. I just took the readMe and updated it a bit

File: apps/docs/pages/guides/auth/auth-helpers/flutter-auth-ui.mdx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+Flutter Auth UI is a Flutter package containing pre-built widgets for authenticating users.
+It is unstyled and can match your brand and aesthetic.
+
+<image src="https://raw.githubusercontent.com/supabase-community/flutter-auth-ui/main/assets/supabase_auth_ui.png" />
+
+## Add Flutter Auth UI
+
+Add the latest version of the package [supabase-auth-ui](https://pub.dev/packages/supabase_auth_ui) to pubspec.yaml:
+
+```bash
```
Comment (dshukertjr) [inline apps/docs/pages/guides/auth/auth-helpers/flutter-auth-ui.mdx:18]:
```suggestion
```yaml
```

File: apps/docs/pages/guides/auth/auth-helpers/flutter-auth-ui.mdx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+dependencies:
+  flutter:
+    sdk: flutter
+  supabase_auth_ui: ^0.1.0+2
+```
+
+### Initialize the Flutter Auth Package
+
+```bash
```
Comment (dshukertjr) [inline apps/docs/pages/guides/auth/auth-helpers/flutter-auth-ui.mdx:27]:
```suggestion
```dart
```

File: apps/docs/pages/guides/auth/auth-helpers/flutter-auth-ui.mdx [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+}
+```
+
+### Email Auth
+
+Use a SupaEmailAuth widget to create an email and password signin and signup form. It also contains a button to toggle to display a forgot password form.
+
+You can pass metadataFields to add additional fields to the form to pass as metadata to Supabase.
+
+```bash
```
Comment (dshukertjr) [inline apps/docs/pages/guides/auth/auth-helpers/flutter-auth-ui.mdx:47]:
```suggestion
```dart
```

File: apps/docs/pages/guides/auth/auth-helpers/flutter-auth-ui.mdx [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+        ),
+    ],
+)
+```
+
+### Magic Link Auth
+
+Use SupaMagicAuth widget to create a magic link signIn form.
+
+```bash
```
Comment (dshukertjr) [inline apps/docs/pages/guides/auth/auth-helpers/flutter-auth-ui.mdx:72]:
```suggestion
```dart
```

File: apps/docs/pages/guides/auth/auth-helpers/flutter-auth-ui.mdx [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+  onSuccess: (Session response) {},
+  onError: (error) {},
+)
+```
+
+### Reset password
+
+Use SupaResetPassword to create a password reset form.
+
+```bash
```
Comment (dshukertjr) [inline apps/docs/pages/guides/auth/auth-helpers/flutter-auth-ui.mdx:84]:
```suggestion
```dart
```

File: apps/docs/pages/guides/auth/auth-helpers/flutter-auth-ui.mdx [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+  onSuccess: (UserResponse response) {},
+  onError: (error) {},
+)
+```
+
+### Phone Auth
+
+Use SupaPhoneAuth to create a phone authentication form.
+
+```bash
```
Comment (dshukertjr) [inline apps/docs/pages/guides/auth/auth-helpers/flutter-auth-ui.mdx:96]:
```suggestion
```dart
```

File: apps/docs/pages/guides/auth/auth-helpers/flutter-auth-ui.mdx [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+  ),
+```
+
+### Social Auth
+
+The package supports login with [official social providers](../../auth#providers).
+
+Use SupaSocialsAuth to create list of social login buttons.
+
+```bash
```
Comment (dshukertjr) [inline apps/docs/pages/guides/auth/auth-helpers/flutter-auth-ui.mdx:109]:
```suggestion
```dart
```

**Review by dshukertjr — COMMENTED** [review]
Thanks @FatumaA ! This is great!

**Review by dshukertjr — APPROVED** [review]

File: apps/docs/pages/guides/auth/auth-helpers/flutter-auth-ui.mdx
Comment (FatumaA) [inline apps/docs/pages/guides/auth/auth-helpers/flutter-auth-ui.mdx:1]: The image here does not render on the page @dshukertjr

**Comment by dshukertjr** [comment]
Hey @gregnr !

I don't have the Vercel permission, but this PR is all set! Could we get it merged?

**Pull request merged**